src/controllers/accountController.js

```diff
@@ -77,17 +77,18 @@ class AccountControl {
 
     if (validCreateAccount.statusCode === 422) return;
```

Comment:
Why are you checking status code.

Reply:
The middleware I am using returns unhandled promise if I dont